spec/services/preserved_object_handler_check_exist_spec.rb

```diff
@@ -318,12 +312,6 @@
                 invalid_po_handler.check_existence
                 expect(invalid_pc.reload.status).to eq PreservedCopy::INVALID_MOAB_STATUS
               end
-              it 'ensures status becomes invalid_moab from expected_vers_not_found_on_storage' do
-                invalid_pc.status = PreservedCopy::EXPECTED_VERS_NOT_FOUND_ON_STORAGE_STATUS
-                invalid_pc.save!
-                invalid_po_handler.check_existence
-                expect(invalid_pc.reload.status).to eq PreservedCopy::INVALID_MOAB_STATUS
-              end
```

Comment:
I think "invalid_moab" should trump most other statuses, if moab validation has already been run.

Reply:
i'd be open to that logic